examples/TestRoutingGuard/can-load.spec.ts

```diff
@@ -0,0 +1,172 @@
+import { Location } from '@angular/common';
+import {
+  Component,
+  inject,
+  Injectable,
+  NgModule,
+  VERSION,
+} from '@angular/core';
+import {
+  CanLoadFn,
+  Router,
+  RouterModule,
+  RouterOutlet,
+} from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+import { from } from 'rxjs';
+import { mapTo } from 'rxjs/operators';
+
+import {
+  MockBuilder,
+  MockRender,
+  NG_MOCKS_GUARDS,
+  NG_MOCKS_ROOT_PROVIDERS,
+  ngMocks,
+} from 'ng-mocks';
+
+// A simple service simulating login check.
+// It will be replaced with its mock copy.
```

Comment:
// It will be replaced with its mock copy. -> // It will be replaced with it's mock copy.